webpack/assets/javascripts/react_app/components/ForemanModal/subcomponents/SubmitOrCancel/SubmitOrCancel.js

```diff
@@ -4,18 +4,35 @@ import PropTypes from 'prop-types';
 import SubmitBtn from './SubmitBtn';
 import CancelBtn from './CancelBtn';
 
-const SubmitOrCancel = ({ submitting, onCancel, onSubmit }) => (
+const SubmitOrCancel = ({
+  submitting,
+  onCancel,
+  onSubmit,
+  submitBtnProps,
+  cancelBtnProps,
+}) => (
   <React.Fragment>
-    <SubmitBtn onSubmit={onSubmit} submitting={submitting} />
+    <SubmitBtn
+      onSubmit={onSubmit}
+      submitting={submitting}
+      {...submitBtnProps}
+    />
     {' ' /* adds whitespace between the buttons */}
```

Comment:
Prefer doing this with CSS (`margin-left` on the rightmost button should do it.)